class.csstidy_optimise.php

```diff
@@ -346,12 +346,14 @@ public function cut_color($color) {
 				if ($color_tmp[$i] > 255)
 					$color_tmp[$i] = 255;
 			}
-			$color = '#';
-			for ($i = 0; $i < 3; $i++) {
-				if ($color_tmp[$i] < 16) {
-					$color .= '0' . dechex($color_tmp[$i]);
-				} else {
-					$color .= dechex($color_tmp[$i]);
+			if (isset($color_tmp[$i])) {
```

Comment:
`$i` is 4 here, coming out from the previous loop. So the test is always false, thus `$color` is always going to be `#` with is not correct

Reply:
ok I have not study all your code, I have just this warning sometime on my log files and this if quickly fix the problem, if the CSS on input is wrong this may come from another lib, I will check, but perhaps you can see for it on the next release.